src/lib/components/HeroComponent.svelte

```diff
@@ -0,0 +1,12 @@
+<div class="flex flex-col flex-wrap items-center pt-24 sm:flex-row">
+    <div class="pb-8 text-center sm:w-2/4 sm:text-left">
+        <h1 class="sm:text-6x1 pb-2 font-dm text-5xl font-bold text-black">
+            Learn. Create. <br /> Innovate.
```

Comment:
We should probs adjust the `max-width` instead of manually breaking the lines.
```suggestion
            Learn. Create. Innovate.
```